coderd/notifications/notifier.go

```diff
@@ -223,6 +230,30 @@ func (n *notifier) prepare(ctx context.Context, msg database.AcquireNotification
 		return nil, xerrors.Errorf("failed to resolve handler %q", msg.Method)
 	}
 
+	appName, err := n.store.GetApplicationName(ctx)
```

Comment:
I think this should get moved into its own function.
You'll then be able to add an internal test to validate that the helpers return the correct thing when the db changes.